content/getting-started/sections/installing-atom.md

```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
 
 #### Installing Atom on Linux
 
-To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. These packages do not currently have auto-update features, so when you would like to upgrade to a new release of Atom, you will have to repeat this installation process.
+To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. If you'd like Atom to be updated when new releases are published, you can configure an official package repository for your distribution.
 
 [releases]: https://github.com/atom/atom/releases/latest
 
 ##### Debian and Ubuntu (deb/apt)
 
-To install Atom on Debian, Ubuntu, or related systems:
+To install Atom on Debian, Ubuntu, or related distributions, add our official
+package repository to your system by running the following commands:
+
+```
+curl -L https://packagecloud.io/AtomEditor/atom/gpgkey | sudo apt-key add -
```

Comment:
There seems to be a convention of adding `$` in front of command lines in this document but I didn't use it in these blocks so it's easier to copy/paste into a terminal.  What do you think?

Reply:
Command line sections should be done with:

``````
``` command-line
$ command <em>replaceable parameter</em>
```
``````

The `$` gets converted to `C:\>` on Windows and the prompt isn't selectable. It's magic :sparkles: